azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultOperationFactory.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.keyvault.spring;
+
+import com.microsoft.azure.keyvault.KeyVaultClient;
+
+public class KeyVaultOperationFactory {
+    public static KeyVaultOperation createDefaultKeyVault(final KeyVaultClient keyVaultClient, final String vaultUri) {
```

Comment:
If this `createDefaultKeyVault` is only used by `test`s, can we move this factory method to `src\test\java` folder? Thank you for your contribution.

Reply:
sure of curse..

Reply:
@PsHype Thanks for the update, left some comments waiting for confirm.